bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetEnergyHandler.java

```diff
@@ -127,15 +138,39 @@ private void subscribeToActivePowerChanges() {
                     }
                 }
             }
-        }, 0, ENERGY_SUBSCRIPTION_PERIOD - 1, TimeUnit.MINUTES);
+        }, 0, POWER_SUBSCRIPTION_PERIOD - 1, TimeUnit.MINUTES);
+    }
+
+    private void subscribeToEnergyTotalizer() {
+        Where w = deviceWhere;
+        if (w == null) {
+            logger.warn("subscribeToEnergyTotalizer() WHERE=null. Skipping");
+            return;
+        }
+
+        energySchedule = scheduler.scheduleWithFixedDelay(() -> {
+            try {
+                send(EnergyManagement.requestCurrentDayTotalizer(w.value()));
+                send(EnergyManagement.requestCurrentMonthTotalizer(w.value()));
+
+            } catch (Exception e) {
+                logger.warn(
+                        "subscribeToEnergyTotalizer() Could not subscribe to totalizers scheduler for WHERE={}. Exception={}",
+                        w, e.getMessage());
+            }
+        }, 0, ENERGY_REFRESH_PERIOD, TimeUnit.MINUTES);
     }
 
     @Override
     public void dispose() {
-        if (notificationSchedule != null) {
-            ScheduledFuture<?> ns = notificationSchedule;
+        if (powerSchedule != null) {
+            ScheduledFuture<?> ns = powerSchedule;
             ns.cancel(false);
```

Comment:
A reason to use false rather than true ?
Same for energySchedule

Reply:
I would  rather set it to true because on dispose() there is need to stop any  running thread: "mayInterruptIfRunning – true if the thread executing this task should be interrupted (if the thread is known to the implementation); otherwise, in-progress tasks are allowed to complete" (javadoc). What do you think about?

Reply:
Yes, we normally use parameter true in dispose().

Reply:
i tried set to false in  dispose() . I tested in   that way  it does't update the item anymore. I would leave it set to false to keep the functionality working, at least for now

Reply:
Strange !